packages/example/src/pages/contributing.mdx

```diff
@@ -0,0 +1,175 @@
+
+1. After it's finished, click on the `Clone or Download` button and copy the contents.
+
+1. In your terminal, clone the repo into your directory of choice
+
+   ```sh
+   git clone [PASTE_URL_HERE]
+   cd gatsby-theme-carbon
+   ```
+
+1. When you clone your forked repo the `origin` is set to your fork by default. You'll want to add a remote that points to the upstream repo.
+
+   ```sh
+   git remote add upstream git@github.com:carbon-design-system/gatsby-theme-carbon.git
+   ```
+
+1. In your terminal, verify the remotes have been set
+
+   ```sh
+   git remote -v
+   ```
+
+## Development
+
+We use [yarn](https://yarnpkg.com/lang/en/docs/install/) and yarn workspaces to develop the Carbon Gatsby theme. This allows us to have a development environment that's closely linked to the theme with minimal setup. Run `yarn install` to install all of the projects dependencies.
+
+This project has two packages: the actual theme package (`gatsby-theme-carbon`) and and the `example` package. The example package emulates a project which uses the theme. It's only dependencies are Gatsby, React and the adjacent theme package. The `example` package also serves as the theme's documentation and [website](https://gatsby-theme-carbon.now.sh); it's deployed on every merge to master.
+
+New theme development will happen in the theme package while documentation and testing of that change will occur through changes in the example directory.
+
+### Development scripts
+
+- `yarn dev` – start the development environment
+- `yarn dev:clean` – clear cache and restart dev
+- `yarn format` – format your code with prettier
+- `yarn lint` - check for errors in your javascript
+
+## Work in a branch
+
+You should always start a new project by pulling upstream changes into master and then creating a new branch. This workflow ensures you don't accidentally commit anything to your master branch and get stuck when trying to open a pull request.
+
+```sh
+git checkout master
+git pull upstream master
+git checkout -b my-branch-name
+```
+
+When you're ready to open a pull request, push to your origin remote.
+
+```sh
+git push origin my-branch-name
+```
+
+You'll get a message in your terminal with a url to open up a pull request in the upstream repository. Navigate to that URL and be sure to give a detailed summary of your pull request in the title and body section of the form.
+
+## Sass and CSS Modules
+
+For internal theme components we use [Sass](https://sass-lang.com/documentation/syntax) and [CSS modules](https://github.com/css-modules/css-modules#css-modules). This allows us to take advantage of the carbon design system resources while not worrying about className collisions. By default, every Sass file will have access to all of the carbon-components [vars](https://github.com/carbon-design-system/carbon/blob/master/packages/components/src/globals/scss/_vars.scss): color, spacing, theme, and motion as well as type and layout mixins are included.
```

Comment:
```suggestion
For internal theme components we use [Sass](https://sass-lang.com/documentation/syntax) and [CSS modules](https://github.com/css-modules/css-modules#css-modules). This allows us to take advantage of the Carbon Design System resources while not worrying about className collisions. By default, every Sass file will have access to all of the Carbon components [vars](https://github.com/carbon-design-system/carbon/blob/master/packages/components/src/globals/scss/_vars.scss): color, spacing, theme, and motion, as well as type and layout mixins, are included.
```

Reply:
@vpicone Take another look at this. How should we reference the Sass vars? carbon-components is deprecated